seacrowd/sea_datasets/fleurs/fleurs.py

```diff
@@ -0,0 +1,292 @@
+"""
+SEA Crowd Data Loader for Fleurs.
+"""
+
+import json
+from itertools import product
+from typing import Dict, List, Tuple
+
+import datasets
+from datasets import load_dataset
+from datasets.download.download_manager import DownloadManager
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import TASK_TO_SCHEMA, Licenses, Tasks
+
+_CITATION = """
+@inproceedings{conneau22_interspeech,
+  author={Alexis Conneau and Ankur Bapna and Yu Zhang and Min Ma and Patrick {von Platen} and Anton Lozhkov and Colin Cherry
+    and Ye Jia and Clara Rivera and Mihir Kale and Daan {van Esch} and Vera Axelrod and Simran Khanuja and Jonathan Clark
+    and Orhan Firat and Michael Auli and Sebastian Ruder and Jason Riesa and Melvin Johnson},
+  title={{XTREME-S: Evaluating Cross-lingual Speech Representations}},
+  year=2022,
+  booktitle={Proc. Interspeech 2022},
+  pages={3248--3252},
+  doi={10.21437/Interspeech.2022-10007}
+}
+"""
+
+logger = datasets.logging.get_logger(__name__)
+
+
+with open(DownloadManager().download_and_extract("seacrowd/sea_datasets/fleurs/lang_config.json"), "r") as f:
```

Comment:
This one refers to a local path, would it still work if we push the dataset to HF?

Reply:
I think as long as the lang_config itself is being pushed to HF repo as well, it still works, bcs it's still located on the same dir level as its dataset script.

You can see it [here](https://huggingface.co/docs/datasets/dataset_script#download-data-files-and-organize-splits) **If the data files live in the same folder or repository of the dataset script, you can just pass the relative paths to the files instead of URLs.**, this should applies to any other files within the repo.

Reply:
@sabilmakbar : Oh, interesting, I just know this. Let's keep this one for now. We can fix it later if it doesn't work as expected when we push it to HF.